gen/dcos-config.yaml

```diff
@@ -4,6 +4,7 @@ package:
       {
         "environment": {
           "MESOS_IP_DISCOVERY_COMMAND": "/opt/mesosphere/bin/detect_ip",
+          "PLATFORM": "{{ platform }}"
```

Comment:
@karlkfi Please add this to the config file rather than the global env: https://github.com/dcos/dcos/blob/master/gen/dcos-config.yaml#L291